### PR TITLE
Enable to switch AWS credentials by option

### PR DESCRIPTION
**Pull request by aibou**
Hi there,

I often use not AWS IAM user credentials but AWS temporary session credentials.
But, this tool does not support switching credentials by command line options.

So I implemented ( ≒ copied from miam ) this feature but please notice that the behaivior of `-s` option changes from `--split` to `--secret-key`

### CODE REVIEW

File: lib/applb/cli.rb [hunk 1]
```diff
@@ -2,6 +2,9 @@
 require 'optparse'
 require 'pathname'
 
+require 'aws-sdk-core'
+Aws.use_bundled_cert!
```
Comment (wata-gh) [inline lib/applb/cli.rb]: Thanks for the PR!

I don't think use_bundled_cert is good to defined in gem.
If people wanted to use bundled cert, they can in other ways.

Please remove this line, or do you have any specific reason to add this line?

**Review by wata-gh — CHANGES_REQUESTED** [review]

File: lib/applb/cli.rb (hunk 1 above)
Comment (aibou) [reply lib/applb/cli.rb]: I understand.
However, this line written in not only this repo but also miam. (https://github.com/codenize-tools/miam/blob/0.2.4/lib/miam.rb#L9)

I think that if this line is removed, the miam code should also be removed, is this right?

Comment (wata-gh) [reply lib/applb/cli.rb]: Yes, I think that it is a good point.
But in other words, **only** miam uses `Aws.use_bundled_cert!` in codenized-tools.

**Comment by fshalor** [comment]
I do not believe this is the right way to do this. The behavior of Codenize-Tools/Maim is perfect. Perhaps that should be merged here?

**Comment by aibou** [comment]
@wata-gh How do you think about this?
I think `Aws.use_bundled_cert!` should be merged for Windows users.
https://docs.aws.amazon.com/sdk-for-ruby/v3/developer-guide/quick-start-guide.html

**Comment by aibou** [comment]
@wata-gh proposed we should give users to choice the option of `Aws.use_bundled_cert!`. I agree with him.
So, I implemented `--use-bundled-cert` option. this enables that choice.

**Review by wata-gh — APPROVED** [review]
The code looks good to me.
So I'll merge this pull-request and fix ci error later.

**Pull request merged**